src/App.tsx

```diff
@@ -1,56 +1,56 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { TabsPage } from './components/TabsPage';
+import { Link, Navigate, Route, Routes, useLocation } from 'react-router-dom';
 
-// const tabs = [
-//   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-//   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-//   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
-// ];
+const HomePage = () => <h1 className="title">Home page</h1>;
 
-export const App = () => (
-  <>
-    {/* Also requires <html class="has-navbar-fixed-top"> */}
-    <nav
-      className="navbar is-light is-fixed-top is-mobile has-shadow"
-      data-cy="Nav"
-    >
-      <div className="container">
-        <div className="navbar-brand">
-          <a href="/" className="navbar-item is-active">
-            Home
-          </a>
-          <a href="/tabs" className="navbar-item">
-            Tabs
-          </a>
-        </div>
-      </div>
-    </nav>
+const NotFoundPage = () => <h1 className="title">Page not found</h1>;
+
+export const App = () => {
+  const location = useLocation();
```

Comment:
```suggestion
  const {pathname} location = useLocation();
```
you can get the pathname right away using the destructuring